toolkits/global/packages/global-corporate-footer/img/logo-springernature_white.svg

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I just whizzed this through SVGOMG. The hbs templates and accompanying config and images etc that you are adding are useful as demos right now - but am I right in thinking that the longer term vision is for hbs templates to be available to component consumers in the future? If so I thought it would make sense to optimise the file size of this SVG now


```<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 175 17"><path fill="#FFF" d="M6.544 6.834c-1.27-.599-2.367-1.115-2.367-2.114 0-.819.529-1.646 1.708-1.646.856 0 1.871.285 3.193.897l1.317-2.64C8.866.435 7.299 0 5.605 0 2.318 0 .356 1.773.356 4.741s2.295 4.113 4.139 5.035c1.338.668 2.493 1.246 2.493 2.334 0 .938-.832 1.568-2.069 1.568-.992 0-2.203-.365-3.601-1.088L0 15.418c1.767 1.007 3.381 1.456 5.226 1.456 3.414 0 5.453-1.9 5.453-5.083 0-3.01-2.293-4.09-4.135-4.957zm11.014-4.935c-1.429 0-3.579 0-4.819.042v14.614h3.626v-4.458h1.102c1.65 0 5.489-.534 5.489-5.486 0-3.039-1.917-4.712-5.398-4.712zm-.114 7.455c-.456 0-.757 0-1.033-.023l-.047-.004V4.665l.047-.003c.321-.022.741-.022 1.077-.022 1.678 0 2.015.969 2.015 2.532.002 1.51-.634 2.182-2.059 2.182zm16.034 1.545.036-.027c1.209-.966 1.875-2.461 1.875-4.209 0-3.513-2.992-4.764-5.793-4.764-1.429 0-3.58 0-4.818.042v14.614h3.626v-4.458h1.102c.22 0 .428-.012.619-.024l.03-.003.016.025c.602.965 1.369 2.519 1.616 4.46h3.967c-.336-2.234-1.552-4.477-2.251-5.619l-.025-.037zm-3.996-1.545c-.456 0-.756 0-1.032-.021l-.047-.004V4.667l.047-.003c.321-.023.741-.023 1.077-.023 1.678 0 2.015.97 2.015 2.534 0 1.507-.635 2.179-2.06 2.179zm8.565 7.201 3.622.002V1.922h-3.622v14.633zm15.115-5.058L48.82 1.921h-3.964v14.633h3.191V7.323l4.503 9.231h3.821V1.921h-3.209v9.576zm11.938-.448h1.509v2.99l-.034.012c-.364.131-1.233.2-1.595.2-1.915 0-2.662-1.419-2.662-5.062 0-3.2 1.102-4.756 3.369-4.756.757 0 1.862.339 3.039.932l.982-2.44c-1.316-.83-3.029-1.325-4.592-1.325-2.198 0-3.782.619-4.844 1.893-1.053 1.265-1.566 3.142-1.566 5.739 0 5.671 1.645 7.779 6.066 7.779 1.408 0 3.528-.366 5.107-1.183V8.554H65.1v2.495zm10.722-.562h4.701V7.658h-4.701V4.773h5.134V1.921h-8.603v12.045c0 .013-.021 1.158.735 1.942.472.491 1.146.741 2.002.746 2.638.015 5.527-.082 5.934-.097v-2.858h-5.203v-3.212h.001zm16.594.45-.024-.038.036-.027c1.209-.966 1.875-2.461 1.875-4.209 0-3.513-2.992-4.764-5.792-4.764-1.429 0-3.58 0-4.818.042v14.614h3.626v-4.458h1.102c.22 0 .427-.012.619-.024l.03-.003.016.025c.602.965 1.369 2.519 1.616 4.46h3.967c-.338-2.233-1.554-4.476-2.253-5.618zm-4.02-1.583c-.456 0-.756 0-1.032-.021l-.047-.004V4.667l.047-.003c.321-.023.741-.023 1.077-.023 1.678 0 2.016.97 2.016 2.534 0 1.507-.635 2.179-2.061 2.179zM109.153 11.595 103.82.15h-4.23v16.405h3.328l.001-11.321 5.292 11.321h4.288V.15h-3.346v11.445zm18.261-6.691h3.267v11.651h3.778V4.904h2.784V1.92h-9.829v2.984zm19.386 6.762c0 1.498-.085 2.415-1.753 2.415-1.599 0-1.869-.752-1.869-2.437V1.921h-3.754v9.92c0 3.491 1.685 5.048 5.463 5.048 3.867 0 5.668-1.771 5.668-5.574V1.921H146.8v9.745zm22.997 2.033v-3.213h4.7V7.657h-4.7V4.772h5.134V1.921h-8.602v12.045c0 .013-.021 1.158.735 1.942.472.491 1.146.741 2.002.746a162.69 162.69 0 0 0 5.934-.097v-2.858h-5.203zm-51.67-11.778-3.544 14.634h3.699l.521-2.324h4.055l.521 2.324h3.901l-3.498-14.634h-5.655zm1.103 9.545 1.507-6.364h.151l1.542 6.364h-3.2zm42.706-.567.035-.027c1.208-.966 1.875-2.461 1.875-4.209 0-3.513-2.992-4.764-5.793-4.764-1.429 0-3.58 0-4.818.042v14.614h3.626v-4.458h1.102c.22 0 .427-.012.619-.024l.03-.003.016.025c.602.965 1.369 2.519 1.617 4.46h3.967c-.336-2.234-1.552-4.477-2.251-5.619l-.025-.037zm-3.995-1.545c-.456 0-.756 0-1.033-.021l-.047-.004V4.667l.047-.003c.321-.023.74-.023 1.077-.023 1.678 0 2.015.97 2.015 2.534.001 1.507-.634 2.179-2.059 2.179z"/></svg>```

Reply:
@benjclark +1 on optimization. I just took the SVG from the global images context folder, though, and added an `id` so I could reference it in the demo JSON. 

Currently, I'm putting SVGs etc in the main folder simply because they are not allowed in the /demo folder. So the lines between demo and consumable code are not very clear. Your thoughts on this would be appreciated. If the `.hbs` is to continue to live in the demo folder, for example, some clear documentation stating it is also a consumable template should be written, I think. This will become clearer to me when we begin integrating these components with springer.com on the SPA team.

Reply:
Yeah I see what you mean. The Frontend Toolkits README specifies that a package structure may contain a `/view` folder for hbs templates presumably with the intention of those being consumable. However Frontend Package Manager just states that demos are generated from the contents of the `/demo` folder. I personally think It would be good if the demos could reference the hbs templates in the `/view` folder so that view templates only need defining once.

I think up until now for whatever reason we have all tended to keep the view templates out of Frontend Toolkits and defined in our apps. But I do agree with the points you raised about this in your previous PR:

> To integrate with toolkits, the SPA team must have .hbs files provided, to be consumable directly. Otherwise the toolkits and the consumer sites/codebases will go out of sync. The provision of demos also makes it possible to test components in isolation (for example, for accessibility) locally, within frontend-toolkits. Hence demos would now have a double purpose.

Perhaps the complexity comes in how we import and use view templates in our applications? It will be interesting to see how you do this with your springer.com work - will it just be a copy and paste job or is there something more sophisticated that could be done? 🤔  I'm not sure really.

Reply:
@benjclark I'm really keen on not just copy/paste because then the consumer code will be severed from the toolkit packages themselves and quickly go out of sync. 

One thing @alexkilgour and I discussed is having the `.hbs` file in the `/view` folder but used as a handlebars partial in the demo. So far, I haven't solved that problem since it will require registering partials for each of the packages and I'm not sure how to go about that yet. Hence, using things as they are already set out for now and putting up with the "demo" name not really being indicative of consumable code 😅.

Reply:
Yes, in that case, I agree with you that the best way forward right now is for our documentation to be updated to make it clear that demo hbs templates are also consumable templates at this point in time.

A discussion around how view templates could be consumed easily and effectively by apps would make a worthwhile frontend-open-space issue, potentially.